ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -503,30 +152,20 @@ sub Configure_Cache {
   my $file_cache = 0; # how much cache is actually configured now in the config file
   my $file_cachemount = '';  # where should the cache be mounted per config file
   my $new_cache = 0;  # in 1k blocks.
-  my $new_cachemount = '';
-  my $automatic_cache = 0;
+  my $file_afsmount = '';
 
   my $ret = 1;
 
   if ($config->elementExists($mypath."/cachesize")) {
     $new_cache = $config->getValue($mypath."/cachesize");  #new cache size
   } else {
-    $self->warn("cannot get CDB $mypath/cachesize, giving up");
+    $self->warn("cannot get CDB $mypath/cachesize - not setting cache size");
     return;
   }
 
-  if ($config->elementExists($mypath."/cachemount")) {
-    my $cdb_cachemount=$config->getValue($mypath."/cachemount"); # new cache mount, if any
-    if ( $cdb_cachemount =~ m;^(/[.\w\/-]+)$; ) {
-      $new_cachemount=$1;
-    } else {
-      $self->warn("CDB new AFS cache mount point $cdb_cachemount looks weird");
-    }
-  }
-
   my $fs_run_cache_string =`fs getcacheparms 2>/dev/null`;
```

Comment:
CAF::Process (with keeps_state?) (and can you check the code for more backticks or `qx//` or `system`? there's no real point in making NoAction effort if you don't use the CAF tools